docs/_articles/en/apex/isv-debugger.md

```diff
@@ -2,10 +2,7 @@
 title: ISV Customer Debugger
 lang: en
 ---
-
-ISV Customer Debugger covers a gap in what you can do with Apex Debugger. As an ISV, you can debug your own code. As a subscriber, you can debug your own code. However, because of the protections against seeing managed code, subscribers can’t debug ISV code in their orgs. With ISV Customer Debugger, an ISV can work with a subscriber to debug issues specific to the subscriber’s org.
-
-An ISV can reproduce issues in the specific environment, so problems can be diagnosed more quickly. You can debug only sandbox orgs.
+Use the ISV Customer debugger, also knowns as ISV debugger if you’re an ISV who needs to work with a subscriber to debug issues specific to the subscriber’s org. The ISV debugger lets an ISV reproduce issues in the subscriber’s environment in a sandbox org for convenient debugging. With the ISV Debugger, an ISV can work with a subscriber to debug issues specific to the subscriber’s org.
```

Comment:
```suggestion
Use the ISV Customer debugger, also known as the ISV debugger, if you’re an ISV who needs to work with a subscriber to debug issues specific to the subscriber’s org. The ISV debugger lets an ISV reproduce issues in the subscriber’s environment in a sandbox org for convenient debugging. With the ISV Debugger, an ISV can work with a subscriber to debug issues specific to the subscriber’s org.
```

Could be helpful to include some of this info as well
```
ISV Customer Debugger covers a gap in what you can do with Apex Debugger. As an ISV, you can debug your own code. As a subscriber, you can debug your own code. However, because of the protections against seeing managed code, subscribers can’t debug ISV code in their orgs. With ISV Customer Debugger, an ISV can work with a subscriber to debug issues specific to the subscriber’s org.
```

Reply:
I updated intro with this text -- 
Salesforce ISVs(Independent Software Vendors) are developers who create managed packages on the Salesforce platform that are available on Salesforce AppExchange. A package subscriber doesn't have access to managed code, which can make debugging challenging. The ISV debugger lets an ISV reproduce issues in the subscriber’s environment in a sandbox org for convenient debugging. With the ISV Debugger, an ISV can work with a subscriber to debug issues specific to the subscriber’s org.